tests/example_netlist.py

```diff
@@ -347,6 +347,13 @@ def main():
     with open(args.physical_netlist, 'wb') as f:
         write_capnp_file(phys_netlist_capnp, f)
 
+    # Test reading back logical and physical netlists
```

Comment:
This isn't actually tested in the CI, this doesn't do what you want.

Reply:
See https://github.com/SymbiFlow/python-fpga-interchange/blob/b26bd256f25b9028a7240189042f4c131f143ee6/tests/test_basics.py#L25

Reply:
Oh right, I see that now, gonna fix it right away